sdk/ai/ai-projects/src/generated/src/parameters.ts

```diff
@@ -103,8 +103,11 @@ export interface ListMessagesQueryParam {
 export type ListMessagesParameters = ListMessagesQueryParam & RequestParameters;
 export type GetMessageParameters = RequestParameters;
 
+export interface UpdateMessageOptions {
```

Comment:
@ganeshyb - I created this to be consistent with having an "options" parameter in the wrapper methods. Is it ok to modify the generated file like this? If so, should I also add this to models.ts so its exported or how should I expose it? If not, how should I go about writing the wrapper? One option is to have a "metadata" parameter of type Record<string, string> | null.

Reply:
Instead of updating the generated code, make the change in the inputOutput.ts file, I am do something similar
`export interface UpdateRunOptions
{
    metadata?: Record<string, string> | null
}`
and in the index.ts
` updateRun: (threadId: string, runId: string, options?: UpdateRunOptions, requestParams?: RequestParameters) =>
      updateRun(context, threadId, runId, {...requestParams, body: options ?? {}}),`